src/test/resources/subscription_schema.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Is this the same as the one in main resources?

Reply:
Now this is deleted and only one file remains in src/main/resources/schemas